src/lib.rs

```diff
@@ -49,5 +49,14 @@ fn main()
     let mut path = unsafe { Path::new(CStr::from_ptr((*info).l_name).to_str().expect("failed to convert to str")) };
     path = path.parent().expect("failed to get parent directory");
 
-    println!("{}", path.to_str().expect("failed to unwrap path"));
+    // this returns the path as jsstring
+    Ok(cx.string(path.to_str().expect("")))
+}
+
+#[neon::main]
+fn main(mut cx: ModuleContext) -> NeonResult<()> {
```

Comment:
don't we need to call `neon_build::setup();` here?

Reply:
According to the official neon documentation you don't need it. 
Probably you need it for larger code bases.